src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -253,7 +256,7 @@ private ResourceContext GetResourceContext(object graph, ODataSerializerContext
             return resourceContext;
         }
 
-        private void WriteDeltaComplexProperties(SelectExpandNode selectExpandNode,
+        internal void WriteDeltaComplexProperties(SelectExpandNode selectExpandNode,
```

Comment:
Document internal method. In addition, the convention is to have internal methods above private methods